Block/BlockContextManagerInterface.php

```diff
@@ -39,6 +39,13 @@ public function addSettingsByType($type, array $settings, $replace = false);
     public function addSettingsByClass($class, array $settings, $replace = false);
 
     /**
+     * @param string $type    Block type to check for
+     *
+     * @return bool
+     */
+    public function exists($type);
```

Comment:
@dbu Ok, I can do that. What should I call it though? "BlockContextManagerExtendedInterface", "BlockContextManagerNewInterface", ...?